include/diff_drive_lib/circular_buffer.hpp

```diff
@@ -9,13 +10,19 @@ class CircularBuffer {
   T* values_;
   size_t size_;
   size_t iter_;
+  std::atomic_bool dynamicAlloc_;
```

Comment:
1. There is no need to use atomic_bool, simple bool is enough. 
2. This could be made const as it is only set upon initialization.
3. Please use lower_case naming convention.
```suggestion
  const bool dynamic_alloc_;
```

Reply:
Done.